src/components/HighContrastSelector/HighContrastSelector.test.tsx

```diff
@@ -0,0 +1,24 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import HighContrastSelector, { HighContrastSelectorProps } from '.';
+
+describe('HighContrastSelector', () => {
+  let component: ShallowWrapper;
+  let props: HighContrastSelectorProps;
+
+  beforeEach(() => {
+    props = {
+      regular: 'regular',
+      highContrast: 'highContrast',
+    };
+
+    component = shallow(<HighContrastSelector {...props} />);
+  });
+
+  describe('render with props', () => {
+    it('hides the highContrast element', () => {
```

Comment:
I'd just say 'renders as expected' here - unfortunately we can't actually determine whether it's hiding/showing any specific component by looking at the snapshot alone.

Reply:
Fixed it.